tests/unit/testsuite.js

```diff
@@ -339,4 +339,23 @@ window.domEqual = function( selector, modifier, message ) {
 	}
 };
 
+// Assertation for checking that an element has multiple classes
+QUnit.assert.hasClasses = function( element, classes ) {
+	var result,
+		classArray = classes.split( " " ),
+		missing = [],
+		found = [];
+
+	classArray.forEach( function( value ) {
+		if ( !element.hasClass( value ) ) {
+			missing.push( value );
+		} else {
+			found.push( value );
+		}
+	});
+	result = !missing.length;
+	QUnit.push( result, classes, result ? classes : found.join( " " ),
```

Comment:
The signature is `result, actual, expected, message`, so the 2nd and 3rd have to switch places, since `classes` represents what we expect, right? And yes, I got that wrong in my original comment where I suggested the custom assertion. Unless I'm wrong here...